eosapi/httpapi/README.md

```diff
@@ -0,0 +1,23 @@
+调用httpapi的方法,可以看client.py这个文件,我下面写的例子
+Call httpapi method, you can see the client.py this file, I wrote the following example.
+```
+# client = Client(['http://eosdapp.oneeos.org:8000'])
+# res1 = client.get_transaction('c6a6733df77c27094f9f65591f9ce783262b73c2a796aec69ea20545e5dce0e8')
+# print(res1)
+# res3 = client.stream_blocks()
+# for item in res3:
+#     print(item)
+# print(res3)
+# client.get_info()
+# client.get_currency_stats('eosio.token','EOS')
+# client.get_code('zhangzhichao')
+# client.get_block(10)
+# client.get_actions('zhangzhichao',1,10)  # need open filter_on params when you run nodeos
+# client.get_currency_balance('eosio.token','zhangzhichao','EOS')
+try:
+    res = client.get_account('zhangzhichao')
+    print(res)
+except Exception as err:
+    print('fail')
+
```

Comment:
we should move the basic Client description into ../../README.md file, so others will see how to use the tool